app/models/host.rb

```diff
@@ -78,6 +82,10 @@ class Host < ApplicationRecord
     )
   }
 
+  scope :joins_test_results, lambda {
```

Comment:
Semantically this is incorrect, should be:
```suggestion
  scope :joins_test_result_profiles, lambda {
```